### PR TITLE
Revert "Yotam playg 39 refine logging"

### PR DESCRIPTION
**Pull request by yotam-yakovw**
Reverts smartFreshBear/RecipeGrabber#102

### CODE REVIEW

File: main_flow/main_flow.py [hunk 1]
```diff
@@ -47,13 +48,17 @@ def loadCache():
     if not (Path(file_instru).is_dir() and Path(file_ingri).is_dir()):
         return None
 
-    main_flow_logger.info('loading cache')
+    logging.info('loading cache')
```
Comment (Hazannovich) [inline main_flow/main_flow.py:51]: You didn't used the logger instance.
(I am not a reviewer I was just interested)

File: main_flow/main_flow.py [hunk 2]
```diff
@@ -9,10 +10,10 @@
 import data_loader
 from training import training_test_cv_divider
 from utils import presistor
-from utils.logger import create_logger_instance
 from stemming import stemming
 
-main_flow_logger = create_logger_instance('Main_Flow')
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s:%(levelname)s:%(message)s')
+logger = logging.getLogger('main_flow')
```
Comment (Hazannovich) [inline main_flow/main_flow.py:16]: Is there a reason behind setting up the logging config in this file or it was like that before?

File: main_flow/main_flow.py (hunk 1 above)
Comment (yotam-yakovw) [reply main_flow/main_flow.py:51]: I'm reverting the pr, the logger instance is deleted and the logging library will be back until the problem is fixed

File: main_flow/main_flow.py (hunk 2 above)
Comment (yotam-yakovw) [reply main_flow/main_flow.py:16]: It was like this before

**Review by smartFreshBear — APPROVED** [review]

**Pull request merged**